ChromeExtension/manifest.json

```diff
@@ -16,7 +16,7 @@
     },
 
     "background": {
-        "scripts": ["js/background.js", "js/jquery.min.js"],
+        "scripts": ["js/globalfunctions.js", "js/background.js", "js/jquery.min.js"],
```

Comment:
Kay so this file not being referenced was what was causing the beahvious assuming an error in the console?